packages/caliper-sawtooth/lib/sawtooth.js

```diff
@@ -365,7 +365,7 @@ class Sawtooth extends BlockchainInterface {
      * @return {Promise} The return promise.
      */
     getContext(name, args) {
-        let config  = require(this.configPath);
+        let config  = require(configPath);
```

Comment:
This one kinda makes me itchy. I think the better solution here would be to set the config inside the Sawtooth constructor, and move the functions into the class so that they have access to a `this.config` property. Moreover, this would remove dynamic requires in the "hot path" during `invokeSmartContract`